ext/bcmath/libbcmath/src/round.c

```diff
@@ -61,7 +96,7 @@ void bc_round(bc_num num, zend_long precision, zend_long mode, bc_num *result)
 	 * If the result of rounding is carried over, it will be added later, so first set it to 0 here.
 	 */
 	if (rounded_len == 0) {
-		*result = bc_copy_num(BCG(_zero_));
+		*result = bc_new_num(1, 0);
```

Comment:
This is not related to this bug, but it was changing a global value and was dangerous, so I fixed it incidentally.